.github/workflows/docs.yaml

```diff
@@ -0,0 +1,47 @@
+name: docs
```

Comment:
```suggestion
name: docs
# Builds the documentation website and publishes it to https://groundlight.github.io
```

As we rely more on GHA, we should get in the habit of writing docstrings at the top for what they do.